bin/node/runtime/src/lib.rs

```diff
@@ -289,50 +293,61 @@ impl pallet_scheduler::Config for Runtime {
     type WeightInfo = ();
 }
 
-parameter_types! {
-    pub const SessionsPerEra: pallet_plasm_rewards::SessionIndex = 6;
-    pub const BondingDuration: pallet_plasm_rewards::EraIndex = 24 * 28;
+pallet_staking_reward_curve::build! {
+    const VALIDATOR_REWARD_CURVE: PiecewiseLinear<'static> = curve!(
```

Comment:
What about moving this as part of the `chain_spec`?

Reply:
not sure about that since it is in the macro